Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/Ride.hs

```diff
@@ -93,7 +94,10 @@ findAllRidesByDriverId ::
   m [Ride]
 findAllRidesByDriverId (Id driverId) = findAllWithKV [Se.Is BeamR.driverId $ Se.Eq driverId]
 
-findActiveByRBId :: MonadFlow m => Id Booking -> m (Maybe Ride)
+findCompletedRideByGHRId :: (MonadFlow m) => Id DDGR.DriverGoHomeRequest -> m (Maybe Ride)
+findCompletedRideByGHRId (Id ghrId) = findOneWithKV [Se.And [Se.Is BeamR.driverGoHomeRequestId $ Se.Eq (Just ghrId), Se.Is BeamR.status $ Se.Eq DRide.COMPLETED]] -- To Be added
```

Comment:
what is to be added ?

Reply:
@arjuna92  @vijaygupta18 KV will not throw error if multiple entries are returned for findOne right ?

Reply:
Secondary key for KV. Its already added forgot to remove the comment.